snapshot/doc/user-guide.md

```diff
@@ -116,7 +116,7 @@ A Persistent Volume will be created and bound to the Persistent Volume Claim. Th
 ## Deleting Snapshot
 A Volume Snapshot `snapshot-demo` can be deleted as shown below:
 ```
-$ kubectl delete -f volumesnapshot/snapshot-demo
+$ kubectl delete volumesnapshot/snapshot-demo
```

Comment:
This is still not correct. Should be `kubectl delete volumesnapshot snapshot-demo`...

Reply:
@tsmetana Its not wrong, above syntax works :)

Reply:
Funny... I never really used it that way. OK then.